### PR TITLE
[Artist/Shows] Fix conditional query for past shows on iPad vs iPhone.

### PR DESCRIPTION
**Pull request by alloy**
Now I determine in objc what device type the app is running on and pass that into RN and the `QueryRenderer` so that it can be used by the shows fragments.

This also adds a recommended extension that will find/highlight `TODO:` and `FIXME:` comments.

Skip New Tests.

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  No CHANGELOG added.
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  Missing Test Files:

- `src/lib/Components/Artist/Shows/__tests__/index-tests.tsx`
- `src/lib/Components/Inbox/Conversations/__tests__/index-tests.tsx`

If these files are supposed to not exist, please update your PR body to include "Skip New Tests".
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 ###### Dev
 
+-   Conditionally use different past artist show fragments for iPad vs iPhone - alloy
+-   Upgrades Relay to ‘modern’ (v1.3.0) - alloy
```
Comment (mzikherman) [inline CHANGELOG.md]: heh

File: Pod/Classes/ViewControllers/ARArtistComponentViewController.m [hunk 2]
```diff
@@ -9,9 +9,10 @@ - (instancetype)initWithArtistID:(NSString *)artistID;
 
 - (instancetype)initWithArtistID:(NSString *)artistID emission:(AREmission *)emission;
 {
+  BOOL isPad = UI_USER_INTERFACE_IDIOM() == UIUserInterfaceIdiomPad;
   if ((self = [super initWithEmission:emission
                            moduleName:@"Artist"
-                    initialProperties:@{ @"artistID": artistID }])) {
+                    initialProperties:@{ @"artistID": artistID, @"isPad": @(isPad) }])) {
```
Comment (mzikherman) [inline Pod/Classes/ViewControllers/ARArtistComponentViewController.m]: should it be `isIpad`? I guess that's not as 'pretty' as `isPad`, something about the capital "I"

**Comment by mzikherman** [comment]
Seems straightforward, just propagating the `isPad` var in the necessary places, as well as the additional query.

Could you point me to where the Obj-C part is that passes this?

File: Pod/Classes/ViewControllers/ARArtistComponentViewController.m (hunk 2 above)
Comment (orta) [reply Pod/Classes/ViewControllers/ARArtistComponentViewController.m]: Yeah, it should, but we went with isPad everywhere inside Eigen for that reason 👍

Comment (alloy) [reply Pod/Classes/ViewControllers/ARArtistComponentViewController.m]: Yeah it’s a bit awkward but it’s what [Apple’s APIs](https://github.com/artsy/emission/pull/746/files/9296b036507c68a5cec6bf6890b0c3a1611bfe4b#diff-263f07bc1b11d93c2b8d4c2e1989281dR12) use and also the naming we’ve always used in Eigen ¯\_(ツ)_/¯

**Comment by alloy** [comment]
> Could you point me to where the Obj-C part is that passes this?

Sure thing, it’s in the `props` dictionary that’s passed to this `initialProperties` parameter https://github.com/artsy/emission/blob/9296b036507c68a5cec6bf6890b0c3a1611bfe4b/Pod/Classes/ViewControllers/ARArtistComponentViewController.m#L15.

The superclass stores it and eventually passes it on to the React Native ‘root view’ which bridges it to the JS runtime and instantiates the `Artist` `AppRegistry` component with those `props`: https://github.com/artsy/emission/blob/bb6c587efbe89191132740a9879b5066ad0ad15d/Pod/Classes/ViewControllers/ARComponentViewController.m#L33

File: Pod/Classes/ViewControllers/ARArtistComponentViewController.m (hunk 2 above)
Comment (alloy) [reply Pod/Classes/ViewControllers/ARArtistComponentViewController.m]: (To be clear, Apple’s API I’m referring to is `UIUserInterfaceIdiomPad`)

Comment (mzikherman) [reply Pod/Classes/ViewControllers/ARArtistComponentViewController.m]: Ah, yes, saw it!

**Pull request merged**